spring-integration-feed/src/main/java/org/springframework/integration/feed/inbound/FeedEntryMessageSource.java

```diff
@@ -165,10 +165,11 @@ private SyndEntry getNextEntry() {
 		if (next == null) {
 			return null;
 		}
-		if (next.getPublishedDate() != null) {
-			this.lastTime = next.getPublishedDate().getTime();
-		}
-		else {
+		
+		Date lastModifiedDate = FeedEntryMessageSource.getLastModifiedDate(next);
+		if (lastModifiedDate != null) {
+			this.lastTime = lastModifiedDate.getTime();
+		} else {
```

Comment:
Sure. Do you have a document that explains preferred coding conventions? Else on new line goes against Java's standard (albeit its and old standard).